### PR TITLE
Single-click to select or open trace

### PR DESCRIPTION
**Pull request by IbrahimFradj**
Related to #317 

I changed the names of the variables example doubleClick to click 
I removed the handleOnExperimentSelected variable from the onClick event because I don't see its use in this event, this variable was not called. 

here is the result.
Animation

### CODE REVIEW

**Comment by MatthewKhouzam** [comment]
Just saw the fix #317 

the commit message could be 

"fix #317 Single-click a trace

If a trace's tab is already open, then a single click simply sets the focus on that tab. If the trace is not open, then a single click should open it.

Signed-off-by: Ibrahim Fradj <ibrahim.fradj@ericsson.com>"

**Comment by ebugden** [comment]
@IbrahimFradj Please also update PR title and description based on MatthewKhouzam's comments. 

Writing clear descriptions of changes is important in software development because:
* **Devs can quickly understand why a design decision was made:** The commit and issue history are usually the only record of the motivation and rationale behind changes. Without writing clear descriptions and associating issues, decisional information can be forgotten or lost if the person leaves the project. Losing decisional information can not only waste a lot of time, but can also lead to technical errors. 
* **You get faster review feedback, good reputation:** Good descriptions makes reviewing code much more comfortable which means you're more likely to get quick feedback and build a good professional reputation among reviewers. 

## Basic PR checklist
- [ ] Concise and complete title (ex. Single-click to select or open trace)
- [ ] Description of changes (What did you change? Why?). For this PR, the title can answer the "What?"
- [ ] Related issues (ex. Fixes #317). For this PR, this answers the "Why?"
- [ ] Recommended: For UI or UX changes, add screenshots or a GIF showing the change.

**Comment by ebugden** [comment]
If found a bug where the trace is opened again in addition to being selected, but I'm not sure if the bug was introduced in these changes. @IbrahimFradj Can you explore what is causing this?

### To reproduce (see video) 
* **0:00** With a trace (A) already open, open a second trace (B) via the File Explorer (Open with --> Trace Viewer). 
* **0:09** After Trace B is open, click Trace A in the "Opened Traces" menu to select it, then click Trace B.
* **0:11** Trace B is opened a 2nd time (instead of just being selected)

https://user-images.githubusercontent.com/28311615/117858670-1db3f380-b25c-11eb-91b3-41f5e3448b0f.mp4

**Review by MatthewKhouzam — DISMISSED** [review]

**Comment by MatthewKhouzam** [comment]
Needs one more reviewer, the only issue, I would squash the two commits together, but as a 1st patch, it's great!

**Review by bhufmann — APPROVED** [review]
LGTM. Thanks!

**Review by MatthewKhouzam — DISMISSED** [review]

**Review by bhufmann — CHANGES_REQUESTED** [review]
Please update the commit message. Use a short description in the first line, "Use single-click to open a trace" and put "Fixes #317" after your long description and before the signed-off line.

**Review by bhufmann — APPROVED** [review]
Looks good to me. Thanks for the contribution.

**Comment by bhufmann** [comment]
There seems to be an issue on GitHub with running some of the build steps. Once that is resolved we can merge this PR.

**Review by MatthewKhouzam — APPROVED** [review]

**Pull request merged**
